http/swagger.yml

```diff
@@ -6672,6 +6672,34 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Authorization"
+    PostBucketRequest:
```

Comment:
👍 I love splitting out Post requests from objects that are returned, can make it a lot more clear what an endpoint is expecting